freqtrade/pairlist/pairlistmanager.py

```diff
@@ -23,24 +23,25 @@ def __init__(self, exchange, config: dict) -> None:
         self._config = config
         self._whitelist = self._config['exchange'].get('pair_whitelist')
         self._blacklist = self._config['exchange'].get('pair_blacklist', [])
-        self._pairlists: List[IPairList] = []
+        self._pairlist_handlers: List[IPairList] = []
```

Comment:
I am not sure i agree with this renaming ...

to me, a pairlist-handler and pairlist-manager is the same (but in other words) ... so now why do i have multiple pairlist handlers?

Reply:
Handlers handle pairlist in the chain, processing (handling) each the pairlist in its manner -- i.e. are those of IPairList type.

The pairlist manager (which is in the same time is the pairlist handler manager) loads handlers (creates the chain) and starts processing the pairlist via the chain of pairlist handlers.

That's how it's used in the docs, approved and merged yesterday.

Reply:
You can find a better name, sure.